README.md

```diff
@@ -1,10 +1,63 @@
-# Generic Project Template
+# SAAS Procurement App 
 
-This repository provides some base files for setting up a repository at
-CDS. Plan is to create more project template for specific technologies:
+The SAAS Procurement app is a Python django project that will aide with the procurement process
 
-- project-template-terraform
-- project-template-python
-- project-template-nodejs
+## Local Development with Containers
 
-Note that default community health files are maintained at https://github.com/cds-snc/.github 
+This project uses [Visual Studio Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers).
+
+Here are the instructions to get started with developing locally.
+
+Requirements:
+
+- Docker installed and running
+- VS Code
+
+Steps:
+
+1. Clone the repo
+2. Open VS Code with Dev Container (see [Quick start: Open an existing folder in a container](https://code.visualstudio.com/docs/remote/containers#_quick-start-open-an-existing-folder-in-a-container))
+3. Go to ```saas-procurement/saas_app```folder
+4. Add a ``.env`` file to the ``/workspace/app`` folder (Contact SRE team for the project specific .env setup)
```

Comment:
should this be `saas-procurement/saas_app`?

Reply:
Good catch Max! I actually re-wrote the README to make it clearer and more consistent with the steps that you actually have to do.